agrirouter-sdk-dotnet-standard-impl/Service/Messaging/Abstraction/SendMultipleMessagesBaseService.cs

```diff
@@ -37,6 +38,19 @@ public MessagingResult Send(SendMultipleMessagesParameters sendMultipleMessagesP
             return _messagingService.Send(messagingParameters);
         }
 
+        /// <summary>
+        ///     Please see base class declaration for documentation.
+        /// </summary>
+        /// <param name="sendMultipleMessagesParameters">-</param>
+        /// <returns>-</returns>
+        public Task<MessagingResult> SendAsync(SendMultipleMessagesParameters sendMultipleMessagesParameters)
```

Comment:
If there is an asynchronous method, there should be a synchronous method as well.

Reply:
Sure. We have a synchronous method public MessagingResult Send(SendMultipleMessagesParameters sendMultipleMessagesParameters)